### DISCUSSION TITLE
unknown packet type encountered: 20

### DISCUSSION THREAD

**Discussion by wojzeh** [bcgit/bc-csharp#404] [Q&A]
Hi there,

Built my C# solution with the recent release of Bouncy Castle, but still getting this error while decrypting a file encrypted with rsa4096 key. Kleopatra decrypts my file with no problem, though.

Any suggestions?

Thank you

**Comment by raphaelcomph-dev** [comment]
Take a look at this https://stackoverflow.com/questions/72145785/pgp-bouncycastle-kleopatra-decryption-unknown-packet-type-20

**Comment by nikhilvc** [comment]
Are there any solutions available for this issue? I'm encountering the same error

**Comment by aramjung** [comment]
I'm facing the error as well.  Are there any updates?

**Comment by jerchap** [comment]
I'm using v2.6.1 and I still get this error.  I wonder if there is anything in the plans for this.

**Reply by rfphill** [reply]
Did you ever resolve this???
